include/deal.II/lac/trilinos_tpetra_vector.h

```diff
@@ -747,21 +742,6 @@ namespace LinearAlgebra
       Tpetra::Vector<Number, int, types::signed_global_dof_index> &
       trilinos_vector();
 
-      /**
-       * Return a const Teuchos::RCP to the underlying Trilinos
-       * Tpetra::Vector class.
-       */
-      Teuchos::RCP<
-        const Tpetra::Vector<Number, int, types::signed_global_dof_index>>
-      trilinos_rcp() const;
```

Comment:
I think we should not remove this function.

I already implemented a preliminary solver for Tpetra, [here](https://github.com/kinnewig/dealii/blob/tpetra_wrapper/source/lac/trilinos_tpetra_solver.cc) and that makes use of this function. 
The solver would become way more complicated without this function.